continuous_mac.sh uses old version of gcloud

`continuous_mac.sh` downloads the latest Cloud SDK but does not actually set the path of gcloud. This can be seen by failures on kokoro-macos such as:

```
testRunAsyncV2Alpha: [ERROR] Failed to execute goal com.google.cloud.tools:appengine-maven-plugin:1.3.2-SNAPSHOT:start (default-cli) on project standard-project: Execution default-cli of goal com.google.cloud.tools:appengine-maven-plugin:1.3.2-SNAPSHOT:start failed: Requires version 171.0.0 or later but found version 159.0.0 -> [Help 1]
```

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -10,9 +10,7 @@ GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
 "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
 
 # Sets gcloud to use the downloaded SDK.
-GCLOUD_PATH=$(which gcloud)
-rm $GCLOUD_PATH
-ln -s "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud $GCLOUD_PATH
+ln -sf "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud $(which gcloud)
 
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -9,14 +9,10 @@ curl https://sdk.cloud.google.com | bash
 GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
 "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
 
-gcloud --version
-
+# Sets gcloud to use the downloaded SDK.
 GCLOUD_PATH=$(which gcloud)
-
 rm $GCLOUD_PATH
 ln -s "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud $GCLOUD_PATH
-
-gcloud --version
 
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -9,6 +9,13 @@ curl https://sdk.cloud.google.com | bash
 GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
 "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
 
+which gcloud
+
+# Remove older versions of gcloud.
+rm -rf /usr/local/Caskroom/google-cloud-sdk
+
+which gcloud
+
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper
 curl  https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.tar.gz -o apache-maven-3.5.0-bin.tar.gz

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -5,7 +5,7 @@ set -e
 # Display commands to stderr.
 set -x
 
-gcloud update components
+gcloud components update
 gcloud components install app-engine-java
 
 gcloud version

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -8,8 +8,6 @@ set -x
 gcloud components update
 gcloud components install app-engine-java
 
-gcloud version
-
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper
 curl  https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.tar.gz -o apache-maven-3.5.0-bin.tar.gz

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -6,7 +6,7 @@ set -e
 set -x
 
 ls .
-readlink -f $(which gcloud)
+readlink $(which gcloud)
 ls /opt
 ls /opt/google-cloud-sdk
 tree /opt

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -9,12 +9,14 @@ curl https://sdk.cloud.google.com | bash
 GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
 "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
 
-which gcloud
+gcloud --version
 
-# Remove older versions of gcloud.
-rm -rf /usr/local/Caskroom/google-cloud-sdk
+GCLOUD_PATH=$(which gcloud)
 
-which gcloud
+rm $GCLOUD_PATH
+ln -s "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud $GCLOUD_PATH
+
+gcloud --version
 
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -7,6 +7,8 @@ set -x
 
 ls .
 readlink -f $(which gcloud)
+ls /opt
+ls /opt/google-cloud-sdk
 tree /opt
 
 curl https://sdk.cloud.google.com | bash

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -8,6 +8,8 @@ set -x
 gcloud update components
 gcloud components install app-engine-java
 
+gcloud version
+
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper
 curl  https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.tar.gz -o apache-maven-3.5.0-bin.tar.gz

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -5,11 +5,10 @@ set -e
 # Display commands to stderr.
 set -x
 
-ls .
+gcloud version
 readlink $(which gcloud)
-ls /opt
-ls /opt/google-cloud-sdk
-tree /opt
+brew cask reinstall google-cloud-sdk
+gcloud version
 
 curl https://sdk.cloud.google.com | bash
 GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -5,17 +5,8 @@ set -e
 # Display commands to stderr.
 set -x
 
-gcloud version
-readlink $(which gcloud)
-brew cask reinstall google-cloud-sdk
-gcloud version
-
-curl https://sdk.cloud.google.com | bash
-GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
-"$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
-
-# Sets gcloud to use the downloaded SDK.
-ln -sf "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud $(which gcloud)
+gcloud update components
+gcloud components install app-engine-java
 
 # temporary workaround until mvn is available in the image by default
 # the integration tests rely on Maven being installed, cannot use the wrapper

--- a/kokoro/continuous_mac.sh
+++ b/kokoro/continuous_mac.sh
@@ -5,6 +5,10 @@ set -e
 # Display commands to stderr.
 set -x
 
+ls .
+readlink -f $(which gcloud)
+tree /opt
+
 curl https://sdk.cloud.google.com | bash
 GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
 "$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java

